hugo/content/overview/pod-affinitiy-overview.md

```diff
@@ -0,0 +1,76 @@
+desired node.  This is specifically done using the following:
+
+- The `preferredDuringSchedulingIgnoredDuringExecution` affinity type, (as defined by the Kubernetes
+API), which defines an anti-affinity rule that Kubernetes will attempt to adhere to, but will not
+guarantee
+- A combination of labels that uniquely identify the pods created by the various deployments 
+listed above as those that the default pod anti-affinity should apply to
+- A topology key of `kubernetes.io/hostname`, which instructs Kubernetes to schedule a pod on
+specific node (i.e. host) only if there is not already another pod in the PG cluster
+scheduled on that same node
+
+While the pod anti-affinity rule discussed above will be enabled by default for all PG clusters,
+it can also be explicitly enabled via the `pgo` CLI by setting the `pod-anti-affinity` option
+to `preferred`:
+
+```bash
+pgo create cluster mycluster --replica-count=2 --pod-anti-affinity=preferred
+```
+
+Or it can also be explicitly enabled globally for all clusters by setting `PodAntiAffinity` to
+`preferred` in the `pgo.yaml` configuration file.
+
+In addition to providing the ability to set default pod anti-affinity using the `preferred`
+option, i.e. an affinity rule that utilizes the `preferredDuringSchedulingIgnoredDuringExecution`
+affinity type, it is also possible to use `requiredDuringSchedulingIgnoredDuringExecution` for
+the default pod anti-affinity, specifically by setting `pod-anti-affinity` to `required`:
+
+```bash
+pgo create cluster mycluster --replica-count=2 --pod-anti-affinity=required
+```
+
+Or similarly the `require` option can be enabled globally for all clusters by setting 
+`PodAntiAffinity` to `require` in the `pgo.yaml` configuration file.
+
+When `require` is utilized for the default pod anti-affinity, a separate node is required for each
+deployment listed above comprising the PG cluster.  This ensures that the cluster remains 
+highly-available by ensuring that node failures do not impact any other deployments in the cluster.
+However, this does mean that the PG primary, each PG replica, the `pgBackRestRepo` and `pgBouncer`,
+will each require a unique node, meaning the minimum number of nodes required for the Kubernetes 
+cluster will increase as more deployments (e.g. PG replicas) are added to the PG cluster.  Further,
+if an insufficient number of nodes are available to support this configuration, certain deployments 
+will fail, since it will not be possible for Kubernetes to successfully schedule the pods for each 
+deployment.
+
+And finally, it is also possible to disable the default pod anti-affinity settings all together.
+Using the `pgo` CLI this is done by setting `pod-anti-affinity` to `disabled`, e.g.:
+
+```bash
+pgo create cluster mycluster --replica-count=2 --pod-anti-affinity=disabled
+```
+
+Or the same setting can be applied globally to all clusters created by setting `PodAntiAffinity` 
+to `require` in the `pgo.yaml` configuration file.
```

Comment:
Should be `disabled` here, if I understand correctly.

Reply:
Changed to `disabled`